packages/node_modules/@node-red/nodes/core/function/10-function.js

```diff
@@ -280,7 +296,9 @@ module.exports = function(RED) {
             }
             var promise = Promise.resolve();
             if (iniScript) {
+                context.send = function(m) { node.send(m); };
                 promise = iniScript.runInContext(context, iniOpt);
+                delete context.send;
```

Comment:
We must not delete `context.send` until the promise returned in the line above is resolved.